gamearea.py

```diff
@@ -215,25 +215,30 @@ def __motion_cb(self, widget, event):
 
     def __press_cb(self, widget, event):
         self.clicked = [event.x, event.y]
+        try:
+            if self.level_data["type"] == GameType.DIVIDED_SCREEN:
+                if self.over_cat is not None:
+                    self.selected_cat = self.over_cat
+                    self.bring_to_front(self.selected_cat)
 
-        if self.level_data["type"] == GameType.DIVIDED_SCREEN:
-            if self.over_cat is not None:
-                self.selected_cat = self.over_cat
-                self.bring_to_front(self.selected_cat)
+                    self.redraw()
 
-                self.redraw()
+            if self.level_data["type"] == GameType.ROWS:
+                self.selected_option = self.over_option
+        except:
```

Comment:
Would be better to catch the particular exception you're trying to catch, same for the except below.